src/tests/ilverify/ILTests/FieldTests.il

```diff
@@ -95,19 +95,8 @@
         ret
     }
 
-    .method public hidebysig instance void 'special.StoreInitonlyFieldOtherInstance..ctor_Invalid_InitOnly'(class FieldTestsType c) cil managed { ret }
-    .method public hidebysig specialname rtspecialname instance void .ctor(class FieldTestsType c) cil managed 
```

Comment:
I deleted this method because it is redundant and body is actually not doing what method name suggests. There are other methods in this file testing `this` (FieldTestsType) type's and `other` type's InstanceInitonlyField.

Reply:
> body is actually not doing what method name suggests.

As far as I can tell, StoreInitonlyFieldOtherInstance described well what the body was doing. Am I missing something?

Reply:
Ops, you are right. I mixed up the "other instance" vs. "other type" instance for which we have StoreInitonlyFieldOtherType.